ios/RNMarigold.m

```diff
@@ -59,95 +41,18 @@ + (BOOL)requiresMainQueueSetup {
     return @[@"inappnotification"];
```

Comment:
Do we still need to support this event in this class?

Reply:
Hi, yes we need to add the supportedEvents. It shows me this error after building when that is not included.
Thread 1: "You must override the `supportedEvents` method of RNMessageStream"

Reply:
Right, we need to implement the method but the `@"inappnotification"` event isn't emitted in this class anymore, only in the `RNMessageStream` class.

Reply:
Oh right. I was looking at the wrong file. Updated the PR. Thanks Ian